app/views/admin/MainAdminView.coffee

```diff
@@ -43,6 +47,7 @@ module.exports = class MainAdminView extends RootView
       @amActually.fetch()
       @supermodel.trackModel(@amActually)
     @featureMode = window.serverSession.featureMode
+    @timeZone = if features?.chinaInfra then 'Asia/Shanghai' else 'America/Los_Angeles'
```

Comment:
since it is an admin view. so maybe poick from browser is not a good idea. but give an option to select timezone maybe better. 
But actually, this `timezone` only uses on when will an activationCode expires. so which timezone doesn't matter much. (most of time we don't want to activationCode really expires. so we will set a large time. so 1 day offset from diff timezone doesn't matter much 😂